framework/wazuh/__init__.py

```diff
@@ -92,6 +95,14 @@ def _initialize(self):
             self.tz_offset = None
             self.tz_name = None
 
+        # UUID info
+        try:
+            uuid_file = os.path.join(self.path, 'api/configuration/security/installation_uid')
+            with open(uuid_file, 'r') as f:
+                self.uuid = f.read().strip()
+        except Exception:
+            self.uuid = None
+
```

Comment:
We can improve this chunk with a more punctual exception like `FileNotFoundError`, or just checking for the existence of the file before opening it.

If you choose the first option, please move the line `uuid_file = os.path.join(self.path, 'api/configuration/security/installation_uid')` outside the `try/except` block.